src/proxy_lib/proxy_lib_new_delete.h

```diff
@@ -0,0 +1,129 @@
+/* ----------------------------------------------------------------------------
+Copyright (c) 2018-2020 Microsoft Research, Daan Leijen
+Copyright (C) 2024 Intel Corporation
+This is free software; you can redistribute it and/or modify it under the
+terms of the MIT license. A copy of the license can be found in the file
+"LICENSE-MIT.txt" in this directory.
```

Comment:
@lplewa could you check if this is ok?

Reply:
I moved the licence to the file, since only one file needs it.

Reply:
Contracts and Licensing Manager confirmed it is OK.